### ISSUE TITLE
[Crash] Engine Crash when creating new would in sample project without Administrator

### ISSUE THREAD

**Issue by 2latemc** [MissingDeadlines/iolite#39]
When the engine is installed into C:\ProgramFiles the sample project is also installed in the same location. When trying to create a new world it probably tries writing to disk (which doesn't work without admin) and then crashes the engine.

**Comment by begla** [comment]
Yes, this is *expected* since IOLITE follows a *portable approach* and requires you to install it in a directory that's writable by the current user. The installer is more of a helper to install the samples and plugins alongside the binary distribution.

This is part of the design philosophy of the engine being the "game" and "editor" in a single executable to allow users to mod games using the same tools used for creation in the first place.

Crashing is always a bad thing, though. So, I'll leave this open and improve handling cases like this in a future build.

Thanks for bringing this to my attention.
